casr/src/triage.rs

```diff
@@ -403,12 +394,13 @@ fn summarize_results(
 
     // Check bad reports.
     if let Ok(err_dir) = fs::read_dir(dir.join("clerr")) {
+        let mut unique = HashSet::new();
         warn!(
             "{} corrupted reports are saved to {:?}",
             err_dir
                 .filter_map(|x| x.ok())
-                .filter_map(|x| x.file_name().into_string().ok())
-                .filter(|x| x.ends_with("casrep"))
+                .map(|x| x.path().with_extension("").with_extension(""))
```

Comment:
What if we have input.png.casrep

Reply:
You may just skip file.gdb.casrep file during counting if file.casrep exists

Reply:
changed